package.json

```diff
@@ -181,6 +181,7 @@
 		"build": "npm run build:packages && wp-scripts build",
 		"build:analyze-bundles": "npm run build -- --webpack-bundle-analyzer",
 		"build:package-types": "node ./bin/packages/validate-typescript-version.js && ( tsc --build || ( echo 'tsc failed. Try cleaning up first: `npm run clean:package-types`'; exit 1 ) ) && node ./bin/packages/check-build-type-declaration-files.js",
+		"build:profile-types": "npm run clean:package-types && node ./bin/packages/validate-typescript-version.js && ( tsc --build --extendedDiagnostics --generateTrace ./ts-traces || ( echo 'tsc failed. Try cleaning up first: `npm run clean:package-types`'; exit 1 ) ) && node ./bin/packages/check-build-type-declaration-files.js && npx @typescript/analyze-trace ts-traces > ts-traces/analysis.txt",
```

Comment:
Currently, if you really want to profile the TS build for slow compilation reasons, the `analysis-txt` file is the most useful one. Should we render a message in the console linking to it? And/or should we spit its contents out to the console as well?

Reply:
For the sake of speed, I tweaked the command so that it prints a success message pointing to the folder and the `analysis.txt` file, plus a command that open that file automatically.

We can always refined as needed in the future?